### PR TITLE
Build page on /kubernetes/resources

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build page on `/kubernetes/resources`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please compare page against [copy doc](https://docs.google.com/document/d/1enaCJ4Vt9a_2_l6xYccnZ-fBVUejPSmfMi8LP2aDA9c/edit)
- Demo at https://ubuntu-com-11544.demos.haus/kubernetes/resources

## Issue / Card

Fixes [#5089](https://github.com/canonical-web-and-design/web-squad/issues/5089)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/kubernetes/resources.html [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/dell-kubernetes-webinar">A decision maker's guide to Kubernetes deployment in your data centre</a></h3>
+      </div>
+      <div class="col-4">
+        <p>How do you compare Kubernetes to other options, get started with containers, and maintain over the long-term?</p>
+        <p>This joint webinar between Canonical and Dell takes a quick snapshot of the container landscape and presents strategies for simple and seamless deployments.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/dell-kubernetes-webinar">
```
Comment (carkod) [inline templates/kubernetes/resources.html:74]: This probably doesn't need absolute URL `/engage/dell-kubernetes-webinar`

File: templates/kubernetes/resources.html [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+          </a>
+        </div>
+      </div>
+    </div>
+  </article>
+  <hr class="is-muted is-fixed-width" />
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/kubernetes-use-cases-webinar">Enterprise Kubernetes use cases: 4 real-world stories</a></h3>
```
Comment (carkod) [inline templates/kubernetes/resources.html:47]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+      </div>
+      <div class="col-4">
+        <p>Despite its increasing popularity, Kubernetes still poses a challenge to enterprises, especially during production deployments. This makes it hard to find success stories and best practices for enterprise Kubernetes adoption – but they do exist.</p>
+        <p>In this webinar, you will learn how different businesses across 4 different industry verticals managed successful Kubernetes deployments using Canonical Kubernetes.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/kubernetes-use-cases-webinar">
```
Comment (carkod) [inline templates/kubernetes/resources.html:55]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+          </a>
+        </div>
+      </div>
+    </div>
+  </article>
+  <hr class="is-muted is-fixed-width" />
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/dell-kubernetes-webinar">A decision maker's guide to Kubernetes deployment in your data centre</a></h3>
```
Comment (carkod) [inline templates/kubernetes/resources.html:66]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+          </a>
+        </div>
+      </div>
+    </div>
+  </article>
+  <hr class="is-muted is-fixed-width" />
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/container-orchestration-edge">Micro clouds: Secure container orchestration at the edge</a></h3>
```
Comment (carkod) [inline templates/kubernetes/resources.html:143]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+      </div>
+      <div class="col-4">
+        <p>The cloud-native way of building software allows for consistency across developer environments and massive scalability of application deployments. Both these attributes are useful at the edge, but create new challenges related to security and resilience.</p>
+        <p>This webinar demonstrates how Canonical's modular technology stack addresses these challenges, using well-known cloud products such as MicroK8s and LXD.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/container-orchestration-edge">
```
Comment (carkod) [inline templates/kubernetes/resources.html:151]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+           </a>
+        </div>
+      </div>
+    </div>
+  </article>
+  <hr class="is-muted is-fixed-width" />
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/edge-kubernetes-raspberry-pi-webinar">K8s at the edge: easy as "Pi"</a></h3>
```
Comment (carkod) [inline templates/kubernetes/resources.html:162]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 8]
```diff
@@ -0,0 +1,490 @@
...
+      </div>
+      <div class="col-4">
+        <p>Discover how to build your own home cloud in this step-by-step video tutorial which combines the power of technologies such as LXD, MicroK8s, Charmed Operators, and Raspberry Pis.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/engage/edge-kubernetes-raspberry-pi-webinar">
```
Comment (carkod) [inline templates/kubernetes/resources.html:169]: Remove https://ubuntu.com

File: templates/kubernetes/resources.html [hunk 9]
```diff
@@ -0,0 +1,490 @@
...
+        <h3 class="p-heading--4"><a href="/engage/atresmedia-charmed-kubernetes-case-study">Atresmedia accesses high availability and unprecedented productivity with Charmed Kubernetes</a></h3>
+      </div>
+      <div class="col-4">
+        <p>Atresmedia, the Spanish OTT media service provider responsible for hits shows such as La Casa de Papel (Money Heist), has been the most loyal customer of Canonical Kubernetes throughout the years.</p>
+        <p>In this case study, you will learn how the company reached its technical and growth acceleration goals by hosting its Atresplayer platform leveraging Charmed Kubernetes.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="/engage/atresmedia-charmed-kubernetes-case-study">
+            <img src="https://ubuntu.com/wp-content/uploads/89d1/Atresmedia.png" alt="Atresmedia accesses high availability and unprecedented productivity with Charmed Kubernetes" width="560" height="302">
```
Comment (carkod) [inline templates/kubernetes/resources.html:193]: Optional comment. Shouldn't we have this in assets?

File: templates/kubernetes/resources.html [hunk 10]
```diff
@@ -0,0 +1,490 @@
...
+    <div class="row">
+      <div class="col-4">
+        <h3 class="p-heading--4"><a href="https://ubuntu.com/blog/kubernetes-ci-cd-pipelines-what-why-and-how">Kubernetes infographic: usage of cloud-native technology in 2021</a></h3>
+      </div>
+      <div class="col-4">
+        <p>A quick, visual summary of the top findings from the Kubernetes and cloud-native operations report.</p>
+      </div>
+      <div class="col-4">
+        <div class="u-hide--small u-align--center">
+          <a href="https://ubuntu.com/blog/kubernetes-ci-cd-pipelines-what-why-and-how">
```
Comment (carkod) [inline templates/kubernetes/resources.html:310]: Remove https://ubuntu.com

**Comment by minkyngkm** [comment]
Thanks @carkod for reviewing. I've updated the urls! :)

File: templates/kubernetes/resources.html (hunk 9 above)
Comment (minkyngkm) [reply templates/kubernetes/resources.html:193]: This is the one Alex provided in the copy doc. You mean we have the same image in the asset? I didn't know that!

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**